src/FluxServiceProvider.php

```diff
@@ -150,6 +151,8 @@ public function boot(): void
         Repeatable::autoDiscover(flux_path('src/Repeatable'), 'FluxErp\Repeatable');
         // Register repeatable artisan commands, jobs and invokable classes (in "Repeatable" directory) from app
         Repeatable::autoDiscover();
+
+        ProductType::register('product', 'flux::livewire.product.product', true);
```

Comment:
I think this doesn't have to be done when running in console?

Reply:
I am not certain about your annotation. What happens in a livewire test which loads the product component?